### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by dsyme**
AppVeyor and Travis are playing up. This is a test PR to sort out the problems independent of the other things in #875

### CODE REVIEW

**Comment by dsyme** [comment]
Added updates to test baselines - will merge this as it is passing Travis and AppVeyor

File: tests/FSharp.Data.DesignTime.Tests/SignatureTests.fs [hunk 1]
```diff
@@ -49,11 +49,16 @@ let normalize (str:string) =
...
 [<TestCaseSource "testCases">]
+[<Test>]
+[<TestCaseSource "testCases">]
 let ``Validate signature didn't change `` (testCase:TypeProviderInstantiation) = 
-    let expected = testCase.ExpectedPath expectedDirectory |> File.ReadAllText |> normalize
-    let output = testCase.Dump resolutionFolder "" runtimeAssembly (*signatureOnly*)false (*ignoreOutput*)false |> normalize
+    let path = testCase.ExpectedPath expectedDirectory 
+    let expected = path |> File.ReadAllText |> normalize
+    let outputRaw = testCase.Dump resolutionFolder "" runtimeAssembly (*signatureOnly*)false (*ignoreOutput*)false |> normalize
+    let output = outputRaw |> normalize
```
Comment (ovatsus) [inline tests/FSharp.Data.DesignTime.Tests/SignatureTests.fs]: Did you meant to call normalize twice?

File: tests/FSharp.Data.DesignTime.Tests/SignatureTests.fs [hunk 2]
```diff
@@ -49,11 +49,16 @@ let normalize (str:string) =
   str.Replace("\r\n", "\n").Replace("\r", "\n").Replace("@\"<RESOLUTION_FOLDER>\"", "\"<RESOLUTION_FOLDER>\"")
 
 [<TestCaseSource "testCases">]
+[<Test>]
+[<TestCaseSource "testCases">]
```
Comment (ovatsus) [inline tests/FSharp.Data.DesignTime.Tests/SignatureTests.fs]: You left a repeated attribute

File: docs/content/library/HtmlProvider.fsx [hunk 3]
```diff
@@ -118,8 +118,9 @@ let doctorWho = new HtmlProvider<"http://en.wikipedia.org/wiki/List_of_Doctor_Wh
 // Get the average number of viewers for each doctor
 let viewersByDoctor = 
     doctorWho.Tables.Overview.Rows 
-    |> Seq.groupBy (fun season -> season.Doctor)
-    |> Seq.map (fun (doctor, seasons) -> doctor, seasons |> Seq.averageBy (fun season -> season.``Average viewers (millions)``))
+    |> Seq.groupBy (fun season -> season.``Doctor(s)``)
+    |> Seq.map (fun (doctor, seasons) -> doctor, seasons |> Seq.averageBy (fun season -> try float season.``Viewers (millions)`` with _ -> 0.0))
```
Comment (ovatsus) [inline docs/content/library/HtmlProvider.fsx]: Filed an issue for this: https://github.com/fsharp/FSharp.Data/issues/877

**Pull request merged**